src/submodules/schedulers.jl

```diff
@@ -211,4 +211,63 @@ function (sched::ByType)(model::ABM)
     return Iterators.flatten(it for it in sched.ids)
 end
 
+"""
+    Schedulers.ByKind(agent_kinds; shuffle_kinds = true, shuffle_agents = true)
+
+A scheduler useful only for mixed agent models using the [`@multiagent`](@ref) macro.
+- `agent_kinds` is a `Tuple` of all valid agent kinds e.g. (:B, :A, :C).
```

Comment:
```suggestion
- `agent_kinds` is a `Tuple` of all valid agent kinds e.g. `(:B, :A, :C)`.
```

when referring to **code** you must use code highlighting.

Reply:
sure, it was just an oversight